Menu/Menu.js

```diff
@@ -1,10 +1,16 @@
-
 const toggleMenu = () => {
-  // Toggle the "menu--open" class on your menu refence. 
+    // Toggle the "menu--open" class on your menu refence. 
 }
 
 // Start Here: Create a reference to the ".menu" class
-const menu;
+const menu = document.querySelector('.menu');
 // create a reference to the ".menu-button" class
-const menuButton;
+const menuButton = document.querySelector('.menu-button');
 // Using your menuButton reference, add a click handler that calls toggleMenu
+
+menuButton.addEventListener("click", (event) => {
+    //console.log('i was clicked.');
+    //console.log(menu.classList);
+    //return menu.style.display = "flex";
+    menu.classList.toggle('display-flex');
```

Comment:
You were supposed to use the `.menu--open {
  display: block;
}` that was already included in the css but you did the same things by adding that display-flex class and using that `display: flex` instead.